packages/trader/src/Modules/CFD/Constants/cfd-specifications.ts

```diff
@@ -1,5 +1,52 @@
 import { localize } from '@deriv/translations';
 
+type TMT5 = {
+    real_synthetic_specs: {
+        [index: string]: string;
+    };
```

Comment:
```suggestion
    real_synthetic_specs: Record<string, string>;
```
Consider using Record utility type
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

Reply:
> 

Cool. Urgently need to apply😎